bin/pycbc_compress_bank

```diff
@@ -125,8 +129,8 @@ else:
 logging.info("getting needed dfs")
 # we'll ensure that the there are atleast 2 samples in a waveform
 seg_lens = 4*numpy.array([max(4./args.sample_rate,
-    2**numpy.ceil(numpy.log2(compress.rough_time_estimate(m1, m2, fmin))))
-    for m1,m2 in zip(templates.mass1, templates.mass2)])
+    2**numpy.ceil(numpy.log2(compress.rough_time_estimate(m1, m2, flow))))
+    for m1,m2,flow in zip(templates.mass1, templates.mass2, templates.f_lower)])
```

Comment:
@cdcapano Yes, that's right. I checked that this works with the O1 bank in which `f_lower` is not stored for the templates. `templates.f_lower` will take up the value for `low_frequency_cutoff` provided to `FilterBank` for that case. If `low_frequency_cutoff` is not provided, and the bank also does not store the cutoffs for the templates, an error is raised when initializing `FilterBank`.